examples/interaction/call_activity_with_tabs_on_click/index.html

```diff
@@ -0,0 +1,80 @@
+        <title>BPMN Visualization - Diagram Navigation</title>
+        <link rel="shortcut icon" href="../../favicon.ico">
+        <link rel="stylesheet" href="../../static/css/spectre.min.css" type="text/css">
+        <link rel="stylesheet" href="../../static/css/icons.min.css" type="text/css">
+        <link rel="stylesheet" href="../../static/css/main.css" type="text/css">
+        <style>
+            .bpmn-container {
+                /* use absolute values for height to ensure that the vertical diagram is not fully displayed when the page is opened. */
+                /* width is relative to the window size */
+                height: 600px;
+                width: 100%;
+                border-style: solid;
+                border-color: #B0B0B0;
+                border-width: 1px;
+                /* This ensures that the parts of the diagram outside of the container are not displayed. */
+                overflow: hidden;
+                background-color: white;
+            }
+        </style>
+    </head>
+    <body>
+        <header class="navbar bg-dark p-2">
+            <section class="navbar-section ml-2">
+                <img src="../../static/img/logo_64x64.ico" alt="logo" class="logo">
+                <span class="h3 text-light ml-2 lib-name-text">BPMN Visualization Examples</span>
+            </section>
+            <section class="navbar-section">
+                <a href="https://process-analytics.github.io/bpmn-visualization-js/" class="mr-2 btn btn-secondary">Docs</a>
+                <div class="dropdown mr-2">
+                    <a href="#" class="btn btn-primary dropdown-toggle" tabindex="0">
+                        GitHub <span class="icon icon-caret icon-in-the-middle"></span>
+                    </a>
+                    <ul class="menu">
+                        <li><a href="https://github.com/process-analytics/bpmn-visualization-js" class="mr-2 btn btn-link">Library</a></li>
+                        <li><a href="https://github.com/process-analytics/bpmn-visualization-examples" class="mr-2 btn btn-link">Examples</a></li>
+                    </ul>
+                </div>
+            </section>
+        </header>
+
+        <section id="section" class="container" style="margin-top: 3rem">
+            <div class="col-11 col-mx-auto">
+                <h2>Switch BPMN diagram display on click on call activity</h2>
+                <p>Click on a call activity, and see what happens 😃</p>
+
+                <ul class="tab bg-secondary">
+                    <li id="main-tab" class="tab-item active">
+                        <a href="#" onclick="openTab('main')">Order Fulfillment</a>
+                    </li>
+                    <li id="secondary-tab" class="tab-item">
+                        <a href="#" onclick="openTab('secondary')">Procurement</a>
+                    </li>
+                </ul>
+                <div id="main-bpmn-container" class="column col-12 col-mx-auto bpmn-container"></div>
```

Comment:
:question: as in https://github.com/process-analytics/bpmn-visualization-examples/pull/108#discussion_r533991081, can we change the cursor to state that the call activity is clickable?